apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/adapters/r4/ClaimAdapter.java

```diff
@@ -0,0 +1,121 @@
+package gov.cms.bfd.server.war.adapters.r4;
+
+import gov.cms.bfd.server.war.adapters.CodeableConcept;
+import gov.cms.bfd.server.war.adapters.Coding;
+import gov.cms.bfd.server.war.adapters.DiagnosisComponent;
+import gov.cms.bfd.server.war.adapters.FhirResource;
+import gov.cms.bfd.server.war.adapters.ItemComponent;
+import gov.cms.bfd.server.war.adapters.ProcedureComponent;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.hl7.fhir.r4.model.Claim;
+
+public class ClaimAdapter implements FhirResource {
```

Comment:
is this supposed to be an abstract class?

Reply:
Nope, it's an adapter, all the adapters just wrap the fhir resources and make their method returns more resource agnostic